recipes/abess/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "abess" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4b3d615efad34f0a24255594b27cec9ff2aae41d2a70812308c4bcceb02b45bb
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build: 
+    - {{ compiler('cxx') }}
+    - python
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - pip
+    - python
+    - pybind11 >=2.6.0
+    - numpy
+  run:
+    - {{ pin_compatible('numpy') }}
+    - scipy 
+    - scikit-learn >=0.24
+    - _openmp_mutex   # [linux]
+    - libstdcxx-ng    # [linux]
+    - libgcc-ng       # [linux]
+    - libcxx          # [osx]
+    - python
+
+test:
+  imports:
+    - abess
+  requires:
+    - pytest
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: "https://abess.readthedocs.io"
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: LICENSE
+  summary: "Fast Best-Subset Selection"
+  description: |
+    abess (Adaptive BEst-Subset Selection) library aims to solve general best subset selection, i.e., 
+    find a small subset of predictors such that the resulting model is expected to have the highest accuracy. 
+    This library implements a generic algorithm framework to find the optimal solution in an extremely fast way. 
+    This framework now supports the detection of best subset under: 
+    linear regression, (multi-class) classification, censored-response modeling, 
+    multi-response modeling (a.k.a. multi-tasks learning), etc. 
+    It also supports the variants of best subset selection like group best subset selection. 
+    Especially, the time complexity of (group) best subset selection for linear regression is certifiably polynomial.
+  doc_url: https://abess.readthedocs.io
+  dev_url: https://github.com/abess-team/abess
+
+extra:
+  recipe-maintainers:
+    - abess-team
```

Comment:
We need approval from the listed co-maintainers that they are ok being listed here. I saw the box was checked in the initial comment on the PR, but just want to confirm that. Also I think we prefer just to list Github users and not organizations, so please remove the `abess-team`.

Reply:
Thanks for this comment! I have removed the `abess-team`.

Reply:
Hi @synapticarbors , @oooo26 and I have approved of being the maintainers.